lib/xmlsec/version.rb

```diff
@@ -1,3 +1,3 @@
 module Xmlsec
-  VERSION = '0.10.3'
```

Comment:
No gem code changed, just running more tests; there's no need to bump the gem version

Reply:
Reverted in https://github.com/instructure/nokogiri-xmlsec-instructure/pull/21/commits/fcf1aa5856cbcbdc0290215f6c04da832e59ba55